.pre-commit-config.yaml

```diff
@@ -5,76 +5,91 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.3.0
     hooks:
-      - id: trailing-whitespace
+      - id: fix-byte-order-marker
+      - id: mixed-line-ending
+        args: [--fix=lf]
```

Comment:
AFAIK this is problematic when working on windows and linux. By default git changes the line endings on checkout on windows to whatever windows is used to (I always forget how it is called) and will change it to what's the default on unix on commit. pre-commit runs before that on commit, so it will always fix line ending on windows.

https://github.com/pre-commit/pre-commit-hooks#mixed-line-ending

Reply:
Ah, fair, I guess we should use auto because of git's `core.autocrlf` (I just disable it, personally).